tests/autoyast/clone.pm

```diff
@@ -26,8 +26,6 @@ sub run {
     assert_script_run 'yast2 --ncurses clone_system', 400;
     upload_logs '/root/autoinst.xml';
 
-    # original autoyast on kernel cmdline
-    upload_logs '/var/adm/autoinstall/cache/installedSystem.xml';
```

Comment:
Why do you want to remove this? It's current way of getting profile which was used for installation and bug for SLE15 is already fixed: https://bugzilla.suse.com/show_bug.cgi?id=1075154

Reply:
I did get test failure due to the bug, but I thought the installedSystem.xml was renamed to "autoinst.ycp" on sle 15. I should treat it carefully for the potential product issue.
And all logs in /var/adm/ will be uploaded in later test module autoyast/logs.pm, seems it is not
required to upload here.
I will discard my change on autoyast/clone.pm 
Thanks for pointing out the issue.